examples/preprocessing/muscle_ica.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-muscle-ica:
+
+==============================
+Removing muscle ICA components
+==============================
+
+Gross movements produce widespread high-frequency activity across all channels
+that is usually not recoverable and so the epoch must be rejected as shown in
+:ref:`ex-muscle-artifacts`. More ubiquitously than gross movements, muscle
+artifact is produced during postural maintenance. This is more appropriately
+removed by ICA otherwise there wouldn't be any epochs left! Note that muscle
+artifacts of this kind are much more pronounced in EEG than they are in MEG.
+
+"""
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+import os.path as op
+import mne
+
+data_path = mne.datasets.sample.data_path()
+raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(raw_fname)
+raw.crop(tmin=100, tmax=130)  # take 30 seconds for speed
+
+# pick only EEG channels, muscle artifact is basically not picked up by MEG
+# if you have a simulatenous recording, you may want to do ICA on MEG and EEG
+# separately
+raw.pick_types(eeg=True)
+
+# ICA works best with a highpass filter applied
+raw.load_data()
+raw.filter(l_freq=1., h_freq=None)
+
+# %%
+# Run ICA
+ica = mne.preprocessing.ICA(
+    n_components=15, method='picard', max_iter='auto', random_state=97)
+ica.fit(raw)
+
+# %%
+# Remove components with postural muscle artifact using ICA
+ica.plot_sources(raw)
+
+# %%
+# By inspection, let's select out the muscle-artifact components based on
+# :footcite:`DharmapraniEtAl2016` manually.
+#
+# The criteria are:
+#   - Positive slope of log-log power spectrum between 7 and 75 Hz
+#     (here just flat because it's not in log-log)
+#   - Peripheral focus or dipole/multi-pole foci (the blue and red
+#     blobs in the topomap are far from the vertex where the most
+#     muscle is)
+#   - Single focal point (low spatial smoothness; there is just one focus
+#     of the topomap compared to components like the first ones that are
+#     more likely neural which spread across the topomap)
+#
```

Comment:
The extra two spaces make it render as a definition list
```suggestion
# The criteria are:
#
# - Positive slope of log-log power spectrum between 7 and 75 Hz
#   (here just flat because it's not in log-log)
# - Peripheral focus or dipole/multi-pole foci (the blue and red
#   blobs in the topomap are far from the vertex where the most
#   muscle is)
# - Single focal point (low spatial smoothness; there is just one focus
#   of the topomap compared to components like the first ones that are
#   more likely neural which spread across the topomap)
#
```